vizro-core/src/vizro/models/_components/form/_text_area.py

```diff
@@ -51,6 +51,6 @@ def build(self):
                     className="text-area",
                 ),
             ],
-            className="input-container",
+            className="form-group",
```

Comment:
@pruthvip15  - I think we should use form-group for the outer container of all inputs that have: label + component